dfastmi/batch/AnalyserAndReporterDflowfm.py

```diff
@@ -378,7 +378,7 @@ def analyse_and_report_dflowfm(
                     units="m",
                 )
         
-        projmesh = outputdir + os.sep + 'projected_mesh.nc'
+        projmesh = str(outputdir.joinpath('projected_mesh.nc'))
```

Comment:
Please use `Path` from `pathlib` library for these kinds of operations.
```Path(outputdir) / "projected_mesh.nc"```

Reply:
heb ik toch gedaan? Joinpath doet dat toch?

Reply:
Did you already discuss this?

Reply:
The `outputdir` argument is typehinted as a string, so please update the type

Reply:
yes you are totally right!

Reply:
I am not doing too much at this code. A lot has changed in DFAST-123. To fully refactor let's do this when we work on DFAST-111 when we refactor the GridEditor calls